src/main/scala/is/hail/methods/PCRelate.scala

```diff
@@ -53,7 +53,7 @@ object PCRelate {
   private val keys = Array("i", "j")
 
   private def toRowRdd(vds: MatrixTable, pcs: DenseMatrix[Double], maf: Double, blockSize: Int, minKinship: Double, statistics: StatisticSubset): RDD[Row] = {
-    val localSampleIds = vds.sampleIds
+    val localSampleIds = vds.stringSampleIds
```

Comment:
Does this depend on string-ness?  Can we just use colKeys?

Reply:
will double check

Reply:
PCRelate hardcodes i and j as strings. It should be relaxed, but that's out of scope here.